crd/src/util.rs

```diff
@@ -508,15 +530,15 @@ mod tests {
     &OpaApi::Data{ package_path: "some_package/some_sub_package".to_string(), rule: "allow".to_string() },
     &OpaApiProtocol::Https,
     None,
-    "https://eebian:8181/v1/data/some_package/some_sub_package/allow"
+    Box::new(["https://", "ebian:8181/v1/data/some_package/some_sub_package/allow"])
     )]
     fn get_connection_string(
         #[case] opa_spec: &str,
         #[case] opa_pods: &str,
         #[case] opa_api: &OpaApi,
         #[case] opa_api_protocol: &OpaApiProtocol,
         #[case] desired_node_name: Option<String>,
-        #[case] expected_result: &str,
+        #[case] expected_result: Box<[&str]>,
```

Comment:
Why all the boxes?

Reply:
For the 4. test case i need to check if it contains https and the correct opa url (because of the introduced randomness). Therefore i need an array and could not make it work unboxed (not sized etc)

Reply:
Ah! In that case it makes more sense to take a slice of string slices:

```suggestion
        #[case] expected_result: &[&str],
```